EnglishTranslation-master/ETC.tsv

```diff
@@ -4265,7 +4265,7 @@ ETC_20150317_004264	Fool (haha)
 ETC_20150317_004265	The amount is low.
 ETC_20150317_004266	Moni
 ETC_20150317_004267	 key has been assigned to 
-ETC_20150317_004268	 to your quick slot.
+ETC_20150317_004268	 on your quick slot bar.
```

Comment:
In game it appears as "2 key has been assigned to Quick Cast on your quick slot bar" (for when it places quick cast in the slot assigned to the 2 key on your keyboard). Optimally it would make way more sense if the skill name and key could be swapped, but I think that's out of scope of the localization.